app/src/preHome.js

```diff
@@ -0,0 +1,163 @@
+        const path = 'listSessions';
+        const myInit = { 
+        //   headers: { 
+        //     Authorization: `Bearer ${(await Auth.currentSession()).getAccessToken().getJwtToken()}`,
+        //   },
+          response: true,
+          queryStringParameters: {
+            PatientId: 'p-3e3477c37d674ecc98e3cdf5487ee07b',
+          },
+        };
+    
+        const result =  await API.get(apiName, path, myInit); 
+        console.log(result);
+        return result;
+    }
+
+    async function listPatients() {
+        const apiName = 'MTADemoAPI';
+        const path = 'listPatients';
+        const myInit = { 
+        //   headers: { 
+        //     Authorization: `Bearer ${(await Auth.currentSession()).getAccessToken().getJwtToken()}`,
+        //   },
+          response: true,
+          queryStringParameters: {
+            PatientId: 'p-3e3477c37d674ecc98e3cdf5487ee07b',
+          },
+        };
+    
+        const result =  await API.get(apiName, path, myInit); 
+        console.log(result);
+        return result;
+    }
+
+    async function listHealthCareProfessionals() {
+        const apiName = 'MTADemoAPI';
+        const path = 'listHealthCareProfessionals';
+        const myInit = { 
+        //   headers: { 
+        //     Authorization: `Bearer ${(await Auth.currentSession()).getAccessToken().getJwtToken()}`,
+        //   },
+          response: true,
+          queryStringParameters: {
+            HealthCareProfessionalId: 'h-67a2c81c20994ff48b52c1b7416667a5',
+          },
+        };
+    
+        const result =  await API.get(apiName, path, myInit); 
+        console.log(result);
+        return result;
+    }
+ 
+    async function createPatient() {
+        const apiName = 'MTADemoAPI';
+        const path = 'createPatient';
+        const myInit = { 
+            //   headers: { 
+            //     Authorization: `Bearer ${(await Auth.currentSession()).getAccessToken().getJwtToken()}`,
+            //   },
+            response: true,
+            queryStringParameters: {
+                'PatientName': 'Red',
+            },
+            body: {
+                'PatientName': 'Red',
+            },
+        };
+    
+        const result =  await API.post(apiName, path, myInit); 
+        console.log(result);
+        return result;
+    }
+
+    async function createHealthCareProfessional() {
+        const apiName = 'MTADemoAPI';
+        const path = 'createHealthCareProfessional';
+        const myInit = { 
+        //   headers: { 
+        //     Authorization: `Bearer ${(await Auth.currentSession()).getIdToken().getJwtToken()}`,
+        //   },
+          response: true,
+          queryStringParameters: {
+            'HealthCareProfessionalName': 'Blue',
+          },
+          body: {
+            'HealthCareProfessionalName': 'Blue',
+          },
+        };
+    
+        const result =  await API.post(apiName, path, myInit); 
+        console.log(result);
```

Comment:
Please remove console logs

Reply:
Done. It's a little bit hard to pin down, but you can search in the file and there will be none.

Reply:
I still do see these : https://github.com/aws-samples/medical-transcription-analysis/blob/storageFeature/frontend/app/src/preHome.js#L48

Reply:
Done.